### PR TITLE
KMS changes

### PR DESCRIPTION
**Pull request by DanSanche**
implements changes suggested in b/113025362 and b/113025156

### CODE REVIEW

File: kms/asymmetric/samples_test.go [hunk 1]
```diff
@@ -146,12 +149,17 @@ func TestRSAEncryptDecrypt(t *testing.T) {
...
 		t.Errorf("ciphertet ending: %s; want: %s", ciphertext[len(ciphertext)-2:], "==")
 	}
-	plaintext, err := decryptRSA(v.ctx, v.client, ciphertext, v.rsaDecryptPath)
+
+	plainBytes, err := decryptRSA(v.ctx, v.client, v.rsaDecryptPath, cipherBytes)
+	if !bytes.Equal(plainBytes, []byte(v.message)) {
+		t.Fatalf("decrypted plaintext does not match input message: want %s, got %s", []byte(v.message), plainBytes)
+	}
+	plaintext := string(plainBytes)
 	if err != nil {
```
[the remark this answers is not in the document]
Comment (ahmetb) [reply kms/asymmetric/samples_test.go]: > Will wait for @ahmetb's approval before submitting.

Not sure why you need my approval for this. I'm not actively looking at kms. Do you just need a golang CR here?

**Comment by ahmetb** [comment]
> Will wait for @ahmetb's approval before submitting.

@tbpg Not sure why you need my approval for this. I'm not actively looking at kms. Do you just need a golang CR here?

**Comment by tbpg** [comment]
> Not sure why you need my approval for this.

I thought your name was on the reviewer list for some reason. So, I figured your review was needed. But, it's not, so never mind!

Should be good to go after the error comment is resolved.

File: kms/asymmetric/samples_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (DanSanche) [reply kms/asymmetric/samples_test.go]: good catch, fixed

**Pull request merged**